src/sync/sync_session.cpp

```diff
@@ -461,7 +461,7 @@ void SyncSession::handle_error(SyncError error)
             // Connection level errors
             case ProtocolError::connection_closed:
             case ProtocolError::other_error:
-            case ProtocolError::pong_timeout:
+//            case ProtocolError::pong_timeout:
```

Comment:
I assume these changes were made in order to build against whichever version of sync you happened to be testing with? We should be sure to revert these and determine the right way to accommodate sync's API changes.